core-nlp-example-master/pom.xml

```diff
@@ -4,28 +4,41 @@
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.techprimers.nlp</groupId>
-    <artifactId>core-nlp-example</artifactId>
+    <groupId>nlp</groupId>
```

Comment:
can you please retain the same group id and artifact id?

Reply:
Sure.